### PR TITLE
Cleanup toolbox and update readme

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues

- https://github.com/cityofaustin/atd-data-tech/issues/18549

## Testing

Review these changes and let me know what you think 🙏 

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

**Review by frankhereford — APPROVED** [review]
# 🚢💨
🧹🧹🧹  This looks great. Thanks for including the hash of the commit to help find them if we ever have to dredge them back up.

**Review by roseeichelmann — APPROVED** [review]
these changes look great to me, thanks for cleaning this up! :sparkles: :broom:

**Review by chiaberry — APPROVED** [review]
Agree with Frank and Rose! 🧹

File: atd-toolbox/download-db-data/Dockerfile
Comment (frankhereford) [inline atd-toolbox/download-db-data/Dockerfile:1]: I absolutely didn't realize this when I approved this PR, but this is a part of the orchestration script, and I believe we're going to need to restore it. I'll confirm and include it in a PR i'm working on now. Thanks!

**Pull request merged**